Kernel/Devices/PATAChannel.cpp

```diff
@@ -275,9 +261,7 @@ bool PATAChannel::ata_read_sectors_with_dma(u32 lba, u16 count, u8* outbuf, bool
 {
     LOCKER(s_lock());
 #ifdef PATA_DEBUG
-    kprintf("%s(%u): PATAChannel::ata_read_sectors_with_dma (%u x%u) -> %p\n",
-        Process::current->name().characters(),
-        Process::current->pid(), lba, count, outbuf);
+    dbg() << Process::current->name().characters() << "(" << Process::current->pid() << "): PATAChannel::ata_read_sectors_with_dma (" << lba << " x" << count << ") -> " << outbuf;
```

Comment:
I replaced `kprintf` with `dbg` call, then I changed the duplicated data in a different commit.
I think it's OK since one can see the logical "steps" towards a complete results in the commits.